setup.cfg

```diff
@@ -1,29 +1,27 @@
-[metadata]
-name = edgepi-rpc-client
-version = 0.0.7
-author = Josiah Kievit, Isaac James
-author_email = jkievit@osensa.com, ijames@osensa.com
-description = Client for EdgePi RPC Server
-long_description = file: README.md
-long_description_content_type = text/markdown
-classifiers = 
-	Programming Language :: Python :: 3
-	License :: OSI Approved :: MIT License
-	Operating System :: OS Independent
-
-[options]
-packages =
-	edgepi_rpc_client
-	edgepi_rpc_client.adc
-	edgepi_rpc_client.dac
-	edgepi_rpc_client.din
-	edgepi_rpc_client.dout
-	edgepi_rpc_client.led
-	edgepi_rpc_client.relay
-	edgepi_rpc_client.tc
-install_requires = 
-	protobuf>=3.20.3
-	edgepi-rpc-protobuf>=1.0.15
-	python-periphery>=2.4.1
-	pyzmq>=25.1.0
-	edgepi-python-sdk>=1.2.6
+[metadata]
+name = edgepi-rpc-client
+version = 0.0.7
+author = Josiah Kievit, Isaac James
+author_email = jkievit@osensa.com, ijames@osensa.com
+description = Client for EdgePi RPC Server
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers = 
+	Programming Language :: Python :: 3
+	License :: OSI Approved :: MIT License
+	Operating System :: OS Independent
+
+[options]
+package_dir=
+    =edgepi_rpc_client/services
+packages=find:
+
+[options.packages.find]
+where=edgepi_rpc_client/services
+install_requires = 
+	protobuf>=3.20.3
+	edgepi-rpc-protobuf>=1.0.15
+	python-periphery>=2.4.1

```

Comment:
are you sure you need to include `python-periphery` ?

Reply:
Did a quick check and it's not needed. Removed.